packages/vscode-extension/src/commands/BaseAction.ts

```diff
@@ -72,8 +76,11 @@ export default abstract class BaseAction {
 		return cliConfigurationService.getProjectFolder(this.cliCommandName);
 	}
 
-	public async run() {
-		this.init();
+	public async run(fsPath?: string) {
+		this.init(fsPath);
+		if(fsPath) {
+			this.filePath = fsPath;
```

Comment:
Will there be a case where filePath will be undefined?

Reply:
Yes, when we are calling importfiles from the command palette.
When calling from the context menu, fsPath would not be undefined.

Anyways, I'm deleting this and adding some logic to the init related to this to make it clear.